libnetwork/drivers/overlay/overlayutils/utils_test.go

```diff
@@ -0,0 +1,82 @@
+package overlayutils
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
+)
+
+func TestAppendVNIList(t *testing.T) {
+	cases := []struct {
+		name    string
+		slice   []uint32
+		csv     string
+		want    []uint32
+		wantErr string
+	}{
+		{
+			name: "NilSlice",
+			csv:  "1,2,3",
+			want: []uint32{1, 2, 3},
+		},
+		{
+			name:    "TrailingComma",
+			csv:     "1,2,3,",
+			want:    []uint32{1, 2, 3},
+			wantErr: `invalid vxlan id value "" passed`,
+		},
+		{
+			name:  "EmptySlice",
+			slice: make([]uint32, 0, 10),
+			csv:   "1,2,3",
+			want:  []uint32{1, 2, 3},
+		},
+		{
+			name:  "ExistingSlice",
+			slice: []uint32{4, 5, 6},
+			csv:   "1,2,3",
+			want:  []uint32{4, 5, 6, 1, 2, 3},
+		},
+		{
+			name:    "InvalidVNI",
+			slice:   []uint32{4, 5, 6},
+			csv:     "1,2,3,abc",
+			want:    []uint32{4, 5, 6, 1, 2, 3},
+			wantErr: "invalid vxlan id value \"abc\" passed",
+		},
+		{
+			name:    "InvalidVNI2",
+			slice:   []uint32{4, 5, 6},
+			csv:     "abc,1,2,3",
+			want:    []uint32{4, 5, 6},
+			wantErr: "invalid vxlan id value \"abc\" passed",
+		},
+	}
+	for _, tt := range cases {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := AppendVNIList(tt.slice, tt.csv)
```

Comment:
While we're writing this with a table test, should we run with `t.Parallel()`?

Reply:
```
ok  	github.com/docker/docker/libnetwork/drivers/overlay/overlayutils	0.006s
```
The tests take so little time to run that the synchronization and scheduling overhead of running them in parallel would at best make it a wash or at worst make things slower.